cmd/aspect/root/root.go

```
@@ -133,5 +89,8 @@ func NewRootCmd(
 		Long:  topics.MustAssetString("tags.md"),
 	})
 
-	return cmd
+	// Run this command after all bazel verbs have been added to "cmd".
+	err := flags.AddBazelFlags(cmd)
```

Move this to outside this constructor so you don't need to return an error.